homeassistant/components/abode/__init__.py

```diff
@@ -351,26 +339,18 @@ def _update_callback(self, device):
 class AbodeAutomation(AbodeEntity):
     """Representation of an Abode automation."""
 
+    _attr_extra_state_attributes = {
+        ATTR_ATTRIBUTION: ATTRIBUTION,
+        "type": "CUE automation",
+    }
```

Comment:
I would not recommend setting a mutable in the class vars in case of a base class.
If one of the instances now modifies it (or in the future) it will affect all instances of this base class.